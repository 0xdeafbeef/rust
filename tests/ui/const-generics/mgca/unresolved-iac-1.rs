#![feature(min_generic_const_args)]
#![feature(inherent_associated_types)]
#![expect(incomplete_features)]

struct A(Box<[u8; Box::b]>);
//~^ ERROR: associated constant `b` not found for

impl A {
  fn c(self) { self.0.d() }
}
fn main() {}
