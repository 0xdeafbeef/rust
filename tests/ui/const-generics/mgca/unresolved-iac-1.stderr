error[E0220]: associated constant `b` not found for `Box<{type error}, {type error}>` in the current scope
  --> $DIR/unresolved-iac-1.rs:5:24
   |
LL |   struct A(Box<[u8; Box::b]>);
   |                          ^ associated item not found in `Box<{type error}, {type error}>`
  --> $SRC_DIR/alloc/src/boxed.rs:LL:COL
  ::: $SRC_DIR/alloc/src/boxed.rs:LL:COL
   |
   = note: associated constant `b` not found for this struct
   |
   = note: the associated constant was found for
           

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0220`.
